src/ethereum_spec_tools/evm_tools/daemon.py

```diff
@@ -59,6 +59,24 @@ def do_POST(self) -> None:
             f"--state.reward={content['state']['reward']}",
         ]
 
+        trace = content.get("trace", False)
+        output_basedir = content.get("output-basedir")
+        if trace:
+            if output_basedir is None:
```

Comment:
```suggestion
            if not output_basedir:
```
Maybe a bit better in case the value is empty string.